modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc

```diff
@@ -22,3 +22,53 @@ Red Hat OpenShift GitOps Operator automatically creates a ready-to-use Argo CD i
 .. Select the *openshift-gitops-cluster* instance to display the password.
 .. Copy the password.
 . Use this password and `admin` as the username to log in to the Argo CD UI in the new window.
+
+The default Argo CD instance installed by the OpenShift GitOps Operator as well as accompanying controllers can now run on the infrastructure nodes of the cluster by setting a simple configuration toggle.
+
+[discrete]
+.Procedure
+. Label the existing nodes:
++
+[source,terminal]
+----
+$ oc label node <node-name> node-role.kubernetes.io/infra=""
+----
++
+.Optional: If required, you can also apply taints and isolate the workloads on infrastructure nodes and prevent other workloads from scheduling on these node:
++
+[source,terminal]
+----
+$ oc adm taint nodes -l node-role.kubernetes.io/infra
+infra=reserved:NoSchedule infra=reserved:NoExecute
+----
+. Add `runOnInfra` toggle in the `GitOpsService` custom resource:
++
+[source,yaml]
+----
+apiVersion: pipelines.openshift.io/v1alpha1
+kind: GitopsService
+metadata:
+  name: cluster
+spec:
+  runOnInfra: true
+----
+. Add `tolerations` to the `GitOpsService` custom resource if taints have been added to the nodes, for example:
++
+[source,yaml]
+----
+  spec:
+    runOnInfra: true
+    tolerations:
+    - effect: NoSchedule
+      key: infra
+      value: reserved
+    - effect: NoExecute
+      key: infra
+      value: reserved
+----
+. Verify that the workloads in the `openshift-gitops` namespace are now scheduled on the infrastructure nodes by viewing `Pods -> Pod details` for any pod in the console UI.
+
+[NOTE]
+====
+Any manually added `nodeSelectors` and `tolerations` added to the default Argo CD custom resource are overwritten by toggle and `tolerations` in the `GitOpsService` custom resource.
```

Comment:
Any manually added `nodeSelectors` and `tolerations` added to the default/Any `nodeSelectors` and `tolerations` manually added to the default
by toggle and/by the toggle and the `tolerations`

Reply:
I didn't understand this, could you elaborate?

Reply:
Maybe this can be rephrased this way?:

Any `nodeSelectors` and `tolerations` manually added to the default Argo CD custom resource will be superseded by the toggle and `tolerations` present in the `GitOpsService` custom resource